### DISCUSSION TITLE
How to set the default font size?

### DISCUSSION THREAD

**Discussion by raarts** [remirror/remirror#1612] [Q&A]
This feels like a stupid question, but I searched, around, and can't find it:

How do I set the default font size? As far as I can see, I can only change font size for selected text.

**Comment by ocavue** [comment]
Adding `font-size` CSS rule to `.remirror-editor` should be the easiest way.

**Reply by raarts** [reply]
This works, but not for my use case. 
I'd like to switch default font size while editing. So with an empty editor, any text I type in should be in 24pt, but then I'd like to switch to 48pt, and any subsequent text I type should by default be in 48pt. Independent of this I'd like to be able to select text, and change font size of the selection. Can this be done?

**Reply by ocavue** [reply]
Is [`@remirror/extension-font-size`](https://remirror.io/docs/extensions/font-size-extension) what you want?

**Reply by raarts** [reply]
I tried that, but as far as I could see, you need to select text first. I couldn't find a way to set a new font size for newly inserted text.

**Reply by ocavue** [reply]
If you only get an empty selection (just a cursor / caret), you still can use `commands.setFontSize(8)` to set the font size.
